src/sample/View.java

```diff
@@ -144,10 +149,16 @@ private void startGameGUI(int parseInt) {
         gameView.setGridSize(parseInt);
         // setting the main game view to a fresh game view
         Main.gameView = gameView;
+        // establishes the actual View for the game which is managed in the View class
+
+        // creation of the grid pane
+        gameView.setupTheGridPane();
+
         // preparation of the game components for movement and treasure hunting
         Main.prepareGame();
-        // establishes the actual View for the game which is managed in the View class
+
         Main.setupView(gameView);
+
         // size based on arg algorithm
```

Comment:
this needs to be more sophisticated and less guesswork. it should ideally be based on the size of the machine's screen.